server/src/auth0/auth0Authentication.js

```diff
@@ -57,34 +57,45 @@ const getGraphcoolUser = (auth0UserId, api) =>
     )
     .then(queryResult => queryResult.User)
 
-//Creates a new User record.
-const createGraphCoolUser = (auth0UserId, profile, api) =>
-  api
+// Creates a new User record.
+const createGraphCoolUser = (auth0UserId, profile, consent, api) => {
+  const savedConsent = {
+    ...consent,
+    method: 'form at profile creation',
+    givenAt: new Date().toISOString()
+  }
+  const profileThatFaqIsAllowedToSave = {
+    name: consent.name ? profile.name : null,
+    picture: consent.picture ? profile.picture : null,
+    email: consent.email ? profile.email : null
+  }
+  return api
     .request(
       `
-        mutation createUser($auth0UserId: String!,
-						$email: String,
-						$name: String,
-						$given_name: String,
-						$family_name: String,
-						$picture: String,
-						$locale: String) {
+        mutation createUser(
+          $auth0UserId: String!,
+          $email: String,
+          $name: String,
```

Comment:
We also need to change every occurence of $given_name and $family_name to use $name.

I don't think $family_name is used, but $given_name is. Maybe we can do `$name.split(' ')[0]` to get it ?

Reply:
No we should not do that. Names are human text and should not be processed in any way. Either we use the name or we use the given name.

Reply:
I agree